### ISSUE TITLE
FS Watching using Notify 

### ISSUE THREAD

**Issue by mohsen1** [napi-rs/node-rs#606]
https://docs.rs/notify/latest/notify/

**Comment by mohsen1** [comment]
Dup of https://github.com/napi-rs/node-rs/issues/570

**Comment by messense** [comment]
I don't think this is a duplicate of #570, #570 is about systemd notify not file system notify.
